CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@ Changelog
 ==========
 
 
+version 0.4.1
```

Comment:
This differently warrants a feature level increase to 0.5.0 following semantic versioining. 

"version develop" is also fine, but given the nature of this PR I will try to release it after merge.